eslint.config.mjs

```diff
@@ -0,0 +1,150 @@
+import globals from 'globals';
+import js from "@eslint/js";
+import mocha from 'eslint-plugin-mocha';
+import pluginChaiFriendly from 'eslint-plugin-chai-friendly'
+import cypress from 'eslint-plugin-cypress/flat';
+import stylisticJs from '@stylistic/eslint-plugin-js';
+
+export default [
+  js.configs.recommended,
+  {
+    name: 'General Linting',
+    languageOptions: {
+      globals: {
+        ...globals.browser,
+        ...globals.node,
+        _PRODUCTION_: true,
+        _DEVELOP_: true,
+        _TEST_: true,
+        _NOW_: true,
+      },
+      ecmaVersion: 2020,
+      sourceType: 'module',
+    },
+    plugins: {
+      '@stylistic/js': stylisticJs
+    },
+    rules: {
+      '@stylistic/js/array-bracket-spacing': ['error', 'never'],
```

Comment:
eslint deprectated rc files and all stylistic rules.  We _could_ switch to Prettier although is more complicated.  In the meantime the stylistic library is not deprecated and compatible.